lib/staccato/adapter/logger.rb

```diff
@@ -5,6 +5,9 @@ module Adapter
     class Logger # The Ruby HTTP Library Adapter
       DEFAULT_FORMATTER = lambda {|params| params.map {|k,v| [k,v].join('=')}.join(' ')}
 
+      attr_reader :uri, :logger
+      attr_accessor :formatter
```

Comment:
I think I'd prefer not to expose this. It should be passed in on `new` and not really changed later. If someone requests this down the line we can reconsider.

Reply:
I generally agree.  I _think_ I kept it r/w simply because stdlib's `Logger#formatter` is an `attr_accessor`.  And it (very slightly) simplified the spec setup.

Reply:
👍🏼